kytos/core/controller.py

```diff
@@ -256,18 +256,22 @@ def start(self, restart=False):
         # pylint: disable=broad-except
         try:
             if self.options.database:
-                self.db_conn_or_core_shutdown()
+                db_conn_wait(db_backend=self.options.database)
                 self.start_auth()
             if self.options.apm:
-                self.init_apm_or_core_shutdown()
+                init_apm(self.options.apm, app=self.api_server.app)
             if not restart:
                 self.create_pidfile()
             self.start_controller()
+        except (KytosDBInitException, KytosAPMInitException) as exc:
+            message = f"Kytos couldn't start because of {str(exc)}"
+            print(message)
+            sys.exit(message)
```

Comment:
The reason why we ended up with another except here, is that for `(KytosDBInitException, KytosAPMInitException)` we don't have to include the traceback `chain=True` that's needed for the other `Exception` that might happen, for those ones we want a full traceback in the logs.